back/boxtribute_server/routes.py

```diff
@@ -113,3 +113,29 @@ def graphql_explorer():
 @api_bp.get("/public")
 def public():
     return EXPLORER_HTML, 200
+
+
+@app_bp.get(f"{CRON_PATH}/<job_name>")
+def cron(job_name):
+    authorized = False
+    try:
+        # https://cloud.google.com/appengine/docs/flexible/scheduling-jobs-with-cron-yaml#securing_urls_for_cron
+        authorized = request.headers["X-Appengine-Cron"] == "true"
+    except KeyError:
+        pass
+    if not authorized:
+        return jsonify({"message": "unauthorized"}), 401
```

Comment:
if a 40X error is thrown, are we automatically notified in slack

Reply:
added emission of sentry warning